builder/vsphere/iso/step_create.go

```diff
@@ -76,8 +76,6 @@ type CreateConfig struct {
 	// here](https://code.vmware.com/apis/358/vsphere/doc/vim.vm.GuestOsDescriptor.GuestOsIdentifier.html)
 	// for a full list of possible values.
 	GuestOSType string `mapstructure:"guest_os_type"`
-	// Set the Firmware at machine creation. Supported values: `bios`, `efi` or `efi-secure`. Defaults to `bios`.
```

Comment:
@azr  `Defaults to bios` was already in the docs here. It was just inconsistent with the HardwareConfig one.

Reply:
Ah yes ! Totally it's less confusing like this !